source/AI.cpp

```diff
@@ -1773,16 +1775,17 @@ void AI::DoScatter(Ship &ship, Command &command)
 
 
 
-Point AI::StoppingPoint(const Ship &ship, bool &shouldReverse)
+// Instead of coming to a full stop, adjust to a target velocity vector
+Point AI::AdjustPoint(const Ship &ship, const Point &targetVelocity, bool &shouldReverse)
```

Comment:
I do think just keeping the StoppingPoint() name would be clearest, even though it technically isn't coming to an absolute stop anymore.